### PR TITLE
Update custom field data table

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 1]
```diff
@@ -266,6 +266,49 @@ To specify another folder, you must change the configuration :
                         namespace_prefix: "My\\AppBundle"
                         path: @AppBundle/Ressources/config/mymapping
 
+Custom search field of columns
+------------------------------
+
+By default, Open Orchestra provides differents search field for columns (text, date, number and boolean). You can specify the types of fields in data attribute ``input-header`` in link of navigation panel (futher information in document of `navigation panel`_).
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: provides differents search fieldS
specify the field types used for a specific datatble in the data attribute `input-header` of the navigation panel link

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 2]
```diff
@@ -266,6 +266,49 @@ To specify another folder, you must change the configuration :
...
+
+To create a custom search field, you should create an backbone view and add it in the ``tableFieldViewconfigurator``.
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: a backbone view dedicated to your search field

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 3]
```diff
@@ -266,6 +266,49 @@ To specify another folder, you must change the configuration :
...
+
+For instance with the field text:
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: the text field

File: en/developer_guide/entity_list_ajax_pagination.rst (hunk 1 above)
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: You should illustrate this with a basic example including 2 or 3 fields and showing what to add and where

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 4]
```diff
@@ -266,6 +266,49 @@ To specify another folder, you must change the configuration :
...
+    )
+
+    ((tableFieldViewconfigurator) ->
+      tableFieldViewconfigurator.text = TextFieldSearchView
+      return
+    ) window.tableFieldViewconfigurator = window.tableFieldViewconfigurator or {}
+
+    You can extend ``AbstractSearchFieldView`` to facilitate research and insertion of the field in the header.
+
+    The key in window.tableFieldViewconfigurator defines the name used in data-attribute (``input-header``).
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: You could mention that it is possible to create a completly dynamic field, not only static ones. By sending an ajax request in the view, the server can interact with the rendering of the search field.

**Comment by alavieille** [comment]
@itkg-ngilain

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 5]
```diff
@@ -246,8 +246,8 @@ The ``open_orchestra_api.annotation_search_reader`` will extract the ``descripti
     $mapping = $this->get('open_orchestra_api.annotation_search_reader')
         ->extractMapping('OpenOrchestra\ModelBundle\Document\Redirection');
 
- Notes
-------
+Notes
+-----
 
 In mapping :
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: `In the mapping` maybe

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 6]
```diff
@@ -266,6 +266,51 @@ To specify another folder, you must change the configuration :
                         namespace_prefix: "My\\AppBundle"
                         path: @AppBundle/Ressources/config/mymapping
 
+Custom search field of columns
+------------------------------
+
+By default, Open Orchestra provides differents search fields for columns (text, date, number and boolean). You can specify the fields types in data attribute ``input-header`` in the navigation panel link (futher information in document of `navigation panel`_).
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: Could you split this sentence in multiple lines ?

Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: in the data attribute

Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: of the navigation panel link

**Comment by alavieille** [comment]
@itkg-ngilain @nicolasThal

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 7]
```diff
@@ -266,6 +266,55 @@ To specify another folder, you must change the configuration :
...
+        )
+        @insertFieldInHeader()
+    )
+
+    ((tableFieldViewconfigurator) ->
+      tableFieldViewconfigurator.text = TextFieldSearchView
+      return
+    ) window.tableFieldViewconfigurator = window.tableFieldViewconfigurator or {}
+
+    You can extend ``AbstractSearchFieldView`` to facilitate research and insertion of the field in the header.
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: do not indent the text :)

**Comment by alavieille** [comment]
@nicolasThal

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 8]
```diff
@@ -266,6 +266,55 @@ To specify another folder, you must change the configuration :
...
+    )
+
+    ((tableFieldViewconfigurator) ->
+      tableFieldViewconfigurator.text = TextFieldSearchView
+      return
+    ) window.tableFieldViewconfigurator = window.tableFieldViewconfigurator or {}
+
+  You can extend ``AbstractSearchFieldView`` to facilitate research and insertion of the field in the header.
+
+  The key in window.tableFieldViewconfigurator defines the name used in data-attribute (``input-header``).
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: add the `` around window....

**Comment by nicolasThal** [comment]
:+1: after adding the ``

**Comment by ngilain** [comment]
:+1:

**Comment by nicolasThal** [comment]
do not forget the squash :)

**Pull request merged**